presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1313,7 +1315,19 @@ public SystemSessionProperties(
                         USE_EXTERNAL_PLAN_STATISTICS,
                         "Use plan statistics from external service in query optimizer",
                         featuresConfig.isUseExternalPlanStatistics(),
-                        false));
+                        false),
+                new PropertyMetadata<>(
+                        JSONPATH_EXTRACTION_ENGINE,
+                        format("Set the engine to be used to compile JsonPath expressions and extract JSON values in JSON_EXTRACT functions. Options are %s",
+                                Stream.of(JsonPathExtractionEngine.values())
+                                        .map(JsonPathExtractionEngine::name)
+                                        .collect(joining(", "))),
+                        VARCHAR,
+                        JsonPathExtractionEngine.class,
+                        JsonPathExtractionEngine.DYNAMIC,
```

Comment:
Can you please default this to Presto, so that users need to opt in to this behavior ? I am not sure if there is any difference between Presto and Jayway parser, but if there is it will cause lot of nuanced issues.